src/main/java/com/exasol/projectkeeper/AbstractProjectKeeperMojo.java

```diff
@@ -29,9 +29,12 @@
     @Parameter(property = "modules")
     private List<String> modules;
 
-    @Parameter(property = "excludeFiles")
+    @Parameter(property = "excludedFiles")
     private List<String> excludedFiles;
 
+    @Parameter(property = "excludedPlugins")
+    private List<String> excludedPlugins;
```

Comment:
The search will work more optimal if you use Set instead of list as you use `contains`

Reply:
that's true